app/javascript/components/patient/assessment/actions/LastDateExposure.js

```diff
@@ -226,51 +222,40 @@ class LastDateExposure extends React.Component {
                   ariaLabel="Last Date of Exposure Input"
                   isClearable
                 />
+                {/* <Row className="pt-2">
+                  <Col> */}
```

Comment:
Was this supposed to be uncommented or removed?

Reply:
removed, my bad. I helped @mdube7 with styling and removed extraneous html

Reply:
fixed: https://github.com/SaraAlert/SaraAlert/pull/895/commits/7f93c59be1e1dfac1bc02adb3a0ed63d68c24210